frontends/precinct-scanner/src/screens/insert_ballot_screen.tsx

```diff
@@ -23,7 +23,7 @@ export function InsertBallotScreen({
       {showNoChargerWarning && (
         <Text warning small center>
           <strong>No Power Detected.</strong> Please ask a poll worker to plug
```

Comment:
I don't think this state or `setup_power_page` is ever shown in the new app experience. This copy is better though, so let's use it in `setup_scanner_screen`, which is shown whenever power is disconnected.